### PR TITLE
UI update

### PR DESCRIPTION
**Pull request by AliNasra**
Included the response items for an informative description of any encountered issues.

### CODE REVIEW

File: prediction-polls/frontend/src/Pages/Auth/SignIn/index.jsx [hunk 1]
```diff
@@ -101,9 +107,12 @@ function SignIn() {
       if (response.status === 200) {
```
Comment (enfurars) [inline prediction-polls/frontend/src/Pages/Auth/SignIn/index.jsx:107]: You are sending your request to local host and checking the wrong status code I guess. Will you change them later?

**Review by enfurars — CHANGES_REQUESTED** [review]
It seems you are prompting the user nicely with your changes but there are two issues about the request.

File: prediction-polls/frontend/src/Pages/Auth/SignIn/index.jsx (hunk 1 above)
Comment (AliNasra) [reply prediction-polls/frontend/src/Pages/Auth/SignIn/index.jsx:107]: Changed them now, sir!

Comment (enfurars) [reply prediction-polls/frontend/src/Pages/Auth/SignIn/index.jsx:107]: URL seems nice now but do you get status 200 for successful request?

**Review by selinisik — APPROVED** [review]
LGTM!

**Comment by AliNasra** [comment]
The front-end team has approved the changes. Bon Apetit!

**Pull request merged**